src/content-handlers/iiif/modules/uv-shared-module/BaseExtension.ts

```diff
@@ -669,7 +670,6 @@ export class BaseExtension<T extends BaseConfig> implements IExtension {
   }
 
   private _updateMetric(): void {
-    setTimeout(() => {
```

Comment:
Should the code inside this function be de-indented for consistency now that the wrapper has been deleted?